### DISCUSSION TITLE
how to set token in cookies and use it in loader to fetch data from a separate backend?

### DISCUSSION THREAD

**Discussion by ShourovRoy** [remix-run/remix#1223] [Q&A]
How can I set token in cookies which comes from separate backend. and how to get that token from cookies in loader function to fetch protected data using that token. Please help me

**Comment by sergiodxa** [comment]
You need to use this https://remix.run/docs/en/v1/api/remix#sessions

**Reply by ShourovRoy** [reply]
It's not working. The token is only showing at client side. Inside loader or session.server.ts its not showing. 
I am using a custom separate backend
